dbus/pointer.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// +build !nodbus,cgo
+
+package dbus
+
+import "unsafe"
+
+// Pointer is an unsafe pointer, only prettier :)
+type Pointer unsafe.Pointer
```

Comment:
Hehe, I was actually thinking of something like `Handle` or something like that. But no strong preference.

Reply:
Do you mean, calling it "Handle" instead of Pointer?